lib/jsdom/living/nodes/Document.webidl

```diff
@@ -1,5 +1,6 @@
 [Constructor,
- Exposed=Window]
+ Exposed=Window,
+ LegacyWindowAlias=HTMLDocument]
```

Comment:
See https://github.com/whatwg/dom/pull/551.

Reply:
Note that this might no longer be correct because of https://github.com/whatwg/html/issues/4792.

Reply:
Well whatwg/dom#551 was closed, so this change should be reverted.